appshared/src/test/java/io/reark/rxgithubapp/shared/utils/NullNetworkInstrumentationTest.java

```diff
@@ -45,10 +43,7 @@ public void setUp() throws Exception {
     @Test
     public void testDecorateNetwork_DoesNotChangeTheHttpClient() {
         OkHttpClient okHttpClient = mock(OkHttpClient.class);
-
         instrumentation.decorateNetwork(okHttpClient);
-
```

Comment:
These two line breaks should be left in place. The idea here is to have unit tests structured to Arrance - Act - Assert steps, with each step divided by a line break:
  - Arrange: setup for the test case (creation of the mocked object in this case)
  - Act: running the actual code to be tested (decorateNetwork)
  - Assert: verify the results (no interactions on the mocked object)